packages/mediacenter/kodi/scripts/service-addon-wrapper

```diff
@@ -64,3 +64,48 @@ if [ -f "${SERVICE_FILE}" ] ; then
       ;;
   esac
 fi
+
+if [ ! -d /storage/.cache/kernel-overlays ] ; then
+  mkdir -p /storage/.cache/kernel-overlays
+fi
+
+# kernel-overlay addons built into the image have their
+# files installed in the default /usr/lib/kernel-overlays
+# location, not inside the kodi addon dir
+
+case "${ADDON_PATH}" in
+  /usr/share/kodi/addons/*)
+    OVERLAY_PATH="/usr/lib/kernel-overlays/${ADDON_ID}"
+    ;;
+  *)
+    OVERLAY_PATH="${ADDON_PATH}/kernel-overlay"
+    ;;
+esac
+
+create_overlay_conf() {
+  rm -f "${OVERLAY_CONF}"
```

Comment:
The "rm -f" ensures we are creating a regular file in the directory. Otherwise, if a user/setup-script/... put a symlink in that dir it'll fail.

I've added the function so we later have a single place to touch in case we add support for multiple-overlays-per-addon - in this case we'll probably have to read in some conf file with the user settings and prefix the entries with the absolute path.